.travis.yml

```diff
@@ -2,29 +2,40 @@ language: php
 
 sudo: false
 
-php:
-  - 7.0
-  - 7.1
-  - 7.2
-
 matrix:
   include:
     - php: 7.0
       env: INFECTION=1
     - php: 7.1
       env: INFECTION=1
+    - php: 7.1
+      env: INFECTION=1 SYMFONY_VERSION="^4.0"
     - php: 7.2
       env: INFECTION=1
+    - php: 7.2
+      env: INFECTION=1 SYMFONY_VERSION="^4.0"
 
 cache:
   directories:
-  - $HOME/.composer/cache
+    - $HOME/.composer/cache
 
 before_install:
   - composer selfupdate
+  - |
+    if [ -n "$SYMFONY_VERSION" != "" ]; then
+      composer config --unset platform.php # removes php 7.0.8 platform fake requirement
+    fi
 
 install:
-  - composer install
+  -  |
+   if [ -n "$LATEST_DEPENDENCIES" != "" ]; then
```

Comment:
It should be `SYMFONY_VERSION`, sorry it was my typo. All builds on matrix now load 3.4, this condition is never `true`

Reply:
Whoops.